sqlalchemy-stubs/dialects/postgresql/__init__.pyi

```diff
@@ -1,5 +1,22 @@
 from typing import Any as _AnyType
 
+from .base import \
+    INTEGER as INTEGER, BIGINT as BIGINT, SMALLINT as SMALLINT, VARCHAR as VARCHAR, CHAR as CHAR, \
+    TEXT as TEXT, NUMERIC as NUMERIC, FLOAT as FLOAT, REAL as REAL, INET as INET, CIDR as CIDR, \
+    UUID as UUID, BIT as BIT, MACADDR as MACADDR, MONEY as MONEY, OID as OID, REGCLASS as REGCLASS, \
+    DOUBLE_PRECISION as DOUBLE_PRECISION, TIMESTAMP as TIMESTAMP, TIME as TIME, DATE as DATE, \
+    BYTEA as BYTEA, BOOLEAN as BOOLEAN, INTERVAL as INTERVAL, ENUM as ENUM, TSVECTOR as TSVECTOR, \
+    DropEnumType as DropEnumType, CreateEnumType as CreateEnumType
```

Comment:
TBH I dom't like the `\` style. Could you please update all long imports in this file and below to be like:
```python
from pack.module import (
    ClassOne, ClassTwo, ClassThree,
    OtherStuff, Whatever
)